docs/release-notes/highlights-1.1.0.asciidoc

```diff
@@ -1,2 +1,73 @@
 [[release-highlights-1.1.0]]
 == 1.1.0 release highlights
+
+[float]
+[id="{p}-110-new-and-notable"]
+=== New and notable
+
+New and notable changes in version 1.1.0 of {n}. See <<release-notes-1.1.0>> for the full list of changes.
+
+[float]
+[id="{p}-110-remote-clusters"]
+==== Remote cluster support
+
+Establishing uni-directional connections from ECK-managed Elasticsearch clusters to other Elasticsearch clusters for link:https://www.elastic.co/guide/en/elasticsearch/reference/current/xpack-ccr.html[cross-cluster replication] and link:https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-cross-cluster-search.html[cross-cluster search] purposes is now easy to achieve through a <<{p}-remote-clusters,few additions to the YAML manifest>>. This feature is available for all managed clusters running Elasticsearch 7.6 or higher with an Enterprise or Enterprise trial licence.
```

Comment:
Technically the in-k8s-cluster version of feature is available even before 7.6. But the cross-k8s-cluster variant is only viable with reasonable effort as of 7.6.